lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -53,12 +53,20 @@ public class ConfigService {
     private final SecretsProvider secretsProvider;
     private final ObjectMapper objectMapper = new ObjectMapper();
 
-    public ConfigService(SSMProvider ssmProvider, SecretsProvider secretsProvider) {
+    private final String featureSet;
```

Comment:
I think this becomes unused as you are setting `String featureSet` in the constructor.

Reply:
Apologies my java is rusty, I forgot you had to declare the types when passing it in eventhough already defining it. Ignore my comment